file-formats/edcr/type/zif_aff_edcr_v1.intf.abap

```diff
@@ -0,0 +1,16 @@
+INTERFACE zif_aff_edcr_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Representation Type</p>
+    "! Representation Type (EDCR) v1
```

Comment:
```suggestion
    "! Representation type
```
Please use sentence case in descriptions as described in our [wiki](https://github.com/SAP/abap-file-formats/blob/main/docs/json.md#description).